src/mtool/api/rest/app.py

```diff
@@ -1253,13 +1288,17 @@ def get_arrays(current_user):
                 # convert to format expected by UI
                 a_info = get_mod_array(a_info)
                 a_info['totalsize'] = int(res["result"]["data"]["capacity"])
-                a_info['usedspace'] = int(res["result"]["data"]["used"])
+                if "used" in res["result"]["data"]:
+                    a_info['usedspace'] = int(res["result"]["data"]["used"])
                 a_info['volumecount'] = len(vol_list)
                 a_info["arrayname"] = res["result"]["data"]["name"]
                 a_info["status"] = array["status"]
                 a_info["situation"] = res["result"]["data"]["situation"]
                 a_info["state"] = res["result"]["data"]["state"]
-                a_info["writeThroughEnabled"] = res["result"]["data"]["writeThroughEnabled"]
+                if "writeThroughEnabled" in res["result"]["data"]:
+                    a_info["writeThroughEnabled"] = res["result"]["data"]["writeThroughEnabled"]
+                else:
+                    a_info["writeThroughEnabled"] = True
```

Comment:
I think writeThroughenabled should be false by default

Reply:
Updated the default value to False.